cmd/internal/cli/actions.go

```diff
@@ -72,7 +72,7 @@ func handleOCI(imgCache *cache.Handle, cmd *cobra.Command, u string) (string, er
 	sysCtx := &ocitypes.SystemContext{
 		OCIInsecureSkipTLSVerify:    noHTTPS,
 		DockerInsecureSkipTLSVerify: noHTTPS,
-		DockerAuthConfig:            authConf,
+		DockerAuthConfig:            &authConf,
```

Comment:
a function changed to return a copy of the auth config instead of a pointer to a global one, just to be on the safe side (there's no good reason to require a pointer here, but we are not changing that)